.changes/3.64.16

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
Missed this in the other PR so I'm submitting an update, but these should go in a unique file in the `.changes/nextrelease/` folder.